src/components/toolbar/toolbar.js

```diff
@@ -169,6 +169,7 @@ function mdToolbarDirective($$rAF, $mdConstant, $mdUtil, $mdTheming, $animate) {
 
           element.css($mdConstant.CSS.TRANSFORM, translateY([-y * shrinkSpeedFactor]));
           contentElement.css($mdConstant.CSS.TRANSFORM, translateY([(toolbarHeight - y) * shrinkSpeedFactor]));
+          contentElement.css('margin-bottom', (toolbarHeight - y) * shrinkSpeedFactor);
```

Comment:
You used the same operation / value twice, this can be refactored into a variable.